src/Sentry/buildTransitive/Sentry.targets

```diff
@@ -120,53 +125,88 @@
 
   </Target>
 
-  <!-- Upload symbols to Sentry after the build. -->
-  <Target Name="UploadSymbolsToSentry" AfterTargets="Build;Publish" DependsOnTargets="PrepareSentryCLI"
-    Condition="'$(SentryUploadSymbols)' == 'true' And '$(SentryCLI)' != ''">
-
-    <Message Importance="High"
-      Text="Preparing to upload debug symbols to Sentry for $(MSBuildProjectName) ($(Configuration)/$(TargetFramework))" />
-
-    <ItemGroup>
-      <SentryCLIUploadSymbolType Include="pdb" />
-      <SentryCLIUploadSymbolType Include="portablepdb" />
-      <SentryCLIUploadSymbolType Include="dsym" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'ios'" />
-      <SentryCLIUploadSymbolType Include="dsym" Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'maccatalyst'" />
-    </ItemGroup>
-
-    <Exec Command="$(SentryCLIDebugFilesUploadCommand) @(SentryCLIUploadSymbolType -> '-t %(Identity)', ' ') $(OutputPath)" IgnoreExitCode="true" ContinueOnError="WarnAndContinue">
-      <Output TaskParameter="ExitCode" PropertyName="_SentryCLIExitCode" />
-    </Exec>
+  <!-- Native AOT publishing
+       While '_IsPublishing' looks a bit "private", it's also OK if it suddenly stops working and this step runs anyway.
+       See https://github.com/dotnet/sdk/issues/26324#issuecomment-1169236993 -->
+  <PropertyGroup Condition="'$(PublishDir)' != '' and '$(PublishAot)' == 'true' and '$(_IsPublishing)' == 'true'">
```

Comment:
I see we already have a `CI_PUBLISHING_BUILD` property... I take it this doesn't work because it's only ever set by by our github actions (and we want this to work even in the context of builds on local development machines)?

Reply:
this is a transitive build target, so one that gets triggered during lib-consumer builds.

Reply:
Aha... cunning. I had no idea we were doing that (or that it was even possible - never had a need before). Is that enabled simply by this?
https://github.com/getsentry/sentry-dotnet/blob/d32f05ba46ca3a022224dae039b27f13691ac06c/src/Sentry/Sentry.csproj#L78-L86

I think that's all we need to do right (reading [the docs](https://learn.microsoft.com/en-us/nuget/concepts/msbuild-props-and-targets)).

Reply:
yes, that's it

Reply:
and I've learned about this a couple of weeks ago myself, for the same reason :)